src/libecalc/common/stream.py

```diff
@@ -0,0 +1,26 @@
+from typing import Literal
+
+from libecalc.common.string_utils import to_camel_case
+from libecalc.common.utils.rates import TimeSeriesFloat, TimeSeriesRate
+from pydantic import BaseModel, Extra
+
+
+class StreamCondition(BaseModel):
+    class Config:
+        extra = Extra.forbid
+        alias_generator = to_camel_case
+        allow_population_by_field_name = True
+
+    rate: TimeSeriesRate
+    pressure: TimeSeriesFloat
+    fluid_density: TimeSeriesFloat = None
```

Comment:
Is this for pump only? Or both compressor and pump? May easily lead to confusion if there is a difference. Not sure if it makes sense with a `Fluid` and `FluidProperties` for pump, as it is simpler..but may be that it is the same...or if we should call it `Fluid` wrt compressors since it is "mostly gas"...to be discussed

Reply:
I guess we could have different types of streams, where pump expects a certain type, and compressor another. But I'm hoping we can find a common interface, where the stream itself can handle density etc. That means using streams as input though, so not yet. 

But I agree, to be discussed.